pkg/env/env.go

```diff
@@ -0,0 +1,65 @@
+package env
+
+import (
+	"os"
+	"strconv"
+	"strings"
+)
+
+// GetIntEnv returns the environment value converted to boolean type, or returns the specified fallback value if the variable with the given key is not present.
+func GetBoolEnv(key string, fallback bool) bool {
```

Comment:
These seem like they might be better off in go-commons. Not urgent...

Reply:
Yes, that's how it was intended. I will move this code after merging the PR. 
_(Adding directly to go-commons at the very beginning is not always convenient for work, because sometimes it needs to be changed quickly)_